commons/src/main/java/com/powsybl/commons/json/JsonUtil.java

```diff
@@ -190,12 +188,23 @@ public static void writeOptionalIntegerField(JsonGenerator jsonGenerator, String
         }
     }
 
+    public static void readDoubleArray(List<Double> doubles, JsonParser parser) throws IOException {
```

Comment:
To be discussed. This seems equivalent:
```
return parser.readValueAs(new TypeReference<List<Double>>(){});
```
Is an utility method really necessary?